pkgs/development/tools/misc/cc-tool/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconofHook
+, boost
+, libusb1
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cc-tool";
+  version = "unstable-2020-05-19";
+
+  src = fetchFromGitHub {
+    owner = "dashesy";
+    repo = pname;
+    rev = "19e707eafaaddee8b996ad27a9f3e1aafcb900d2";
+    hash = "sha256:1f78j498fdd36xbci57jkgh25gq14g3b6xmp76imdpar0jkpyljv";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook ];
```

Small nitpick, let's keep this alphabetical:

```suggestion
  nativeBuildInputs = [ autoreconfHook pkg-config ];
```